JS/app.js

```diff
@@ -1,58 +1,123 @@
 'use strict';
-console.log('hello there!');
+// console.log('hello there!');
 // this is my special '42' note!!
 
 
 // global variables
-var hours = ['6am ','7am','8am','9am','10am','11am','12am','13am','14am','15am','16am','17am','18am','19am','20am'];
+var hours = ['6am ','7am','8am','9am','10am','11am','12am','1pm','2pm','3pm','4pm','5pm','6pm','7pm','8pm','9pm'];
 //array storing all cookie values
-var tableEl = this.document.getElementById('first' );
+var tableEl = document.getElementById('first' );
+var allStores = [0];
```

Comment:
This may not be a problem, but it looks like your intent was to create an empty array to push store objects into. You don't need the '0' to be there in order to create it initially.